ext/js/display/display-anki.js

```diff
@@ -370,6 +370,31 @@ export class DisplayAnki {
         }
     }
 
+    /**
+     * @param {HTMLButtonElement} button
+     */
+    _showDuplicateAddButton(button) {
+        const isKanjiAdd = button.dataset.mode === 'term-kanji';
+
+        const title = button.getAttribute('title');
+
+        if (title) {
+            button.setAttribute('title', title.replace(/Add (?!duplicate)/, 'Add duplicate '));
+        }
+
+        const {hotkey} = button.dataset;
+
+        if (hotkey) {
+            button.dataset.hotkey = hotkey.replace(/Add (?!duplicate)/, 'Add duplicate ');
```

Comment:
The hotkey update that is being done here feels kind of like a hack. What would probably be better is to add a utility function to get/set/modify the label text of an element with a hotkey property, somewhere in `HotkeyHelpController`. This would at the very least make the code more clear as to what is going on and what it relates to. Doing a full JSON parse/re-stringify of the attribute is probably safest.

Reply:
gave it a try, but not sure that's exactly what you want